inst/@sym/function_handle.m

```diff
@@ -130,11 +133,15 @@
     M.name = out{1}{1};
     M.code = out{1}{2};
 
-    [fid,msg] = fopen(M.name, 'w');
+    file_to_write = fullfile(fcnpath, [fcnname '.m']);
+    [fid,msg] = fopen(file_to_write, 'w');
     assert(fid > -1, msg)
     fprintf(fid, '%s', M.code)
     fclose(fid);
-    fprintf('Wrote file %s.\n', M.name);
+    fprintf('Wrote file %s.\n', file_to_write);
+    if (exist(fcnpath))
+        addpath(fcnpath);
+    end
     f = str2func(fcnname);
```

Comment:
this is scary: I sort of think its the user's responsbility, if she wants.  Plus added `/tmp` to the path is bound to be some sort of code-injection security bug (Bob create `/tmp/ls.m` which contains bad code for Alice to run etc).